virl/files/mitaka.bashrc

```diff
@@ -120,7 +120,7 @@ export OS_USERNAME=admin
 export OS_PASSWORD=ADMIN_PASS
 export OS_TENANT_NAME=admin
 export OS_AUTH_URL=http://controller:35357/v3
-export OS_SERVICE_ENDPOINT=http://controller:35357/v3
+export OS_SERVICE_ENDPOINT=http://controller:35357/v2.0
```

Comment:
why are you doing this in mitaka version...what is breaking by having this be v3?